### DISCUSSION TITLE
React 19 support

### DISCUSSION THREAD

**Discussion by snovik** [mobxjs/mobx#3984] [Q&A] [answered]
Hello.
Mobx will support react 19 (mobx-react/mobx-react-lite)? If yes, when is it expected?
* Current versions only support up to version 18

**Comment by imjordanxd** [comment, marked as answer]
There has been React 19 support since `mobx-react-lite@4.1.0` and `mobx-react@9.2.0`

Releases: 
- https://github.com/mobxjs/mobx/releases/tag/mobx-react%409.2.0
- https://github.com/mobxjs/mobx/releases/tag/mobx-react-lite%404.1.0
